scripts/create-local-tag-for-release

```diff
@@ -83,17 +83,33 @@ validate_args() {
     exit 1
 }
 
-# create the new tag
+sync_local_tags_with_remote() {
```

Comment:
can you explain this a bit? I thought creating local tag is the first step before creating a remote tag

Reply:
this code vipes out local tags and pulls in all remote tags from upstream. This is done to eliminate the possiblity of local tags being out of sync with remote tags.

remote tag is not created in this script. 

will rename the function to `sync_local_tags_from_remote`